modules/ROOT/partials/integrations/blazor-tech-ref.adoc

```diff
@@ -188,19 +188,37 @@ In your component:
 
 === `LicenseKey`
 
-Specifies the {productname} license key. Required for self-hosted deployments of {productname}. This property is not required for deployments using the {cloudname}. For more information on licensing, see: xref:license-key.adoc[License key].
+Specifies the {productname} license key. Required for self-hosted deployments of {productname}. This property is not required for deployments using the {cloudname}.
+
+License keys must start with the "T8LK:" prefix and use client-side JWT validation that runs entirely in the browser, requiring no server communication or "phone home" checks. For more information, see: xref:license-key.adoc[License Key].
 
 *Type:* `+String+`
 
-==== Example using LicenseKey
+*Possible values:*
+* `null` - Use this when loading from {cloudname} with an API key
```

Comment:
Note in hybrid mode you can both supply a license key and connect to cloud - in which case it will fallback to the locally provided key if cloud is not contactable.